src/settings/SettingsPage.js

```diff
@@ -56,6 +60,26 @@ export const SettingsPage = ({
 
     <Experiments t={t} />
 
+    <Box className='mb3 pa4'>
+      <div className='mb4'>
+        <Title>{t('CLI TUTOR MODE')}</Title>
+        <Checkbox className='dib' onChange={doToggleCliTutorMode} checked={isCliTutorModeEnabled}
+          label={<span className='fw5 f6'>{t('Enable command-line interface (CLI) tutor mode')}</span>}/>
```

Comment:
You need to add a translation key here, not the text

Reply:
Namely, add the key to `public/locales/en/settings.json` 
(no need to add it to other locales, that will happen automatically after PR is merged)

Reply:
@rafaelramalho19 @lidel i've replace these text lines with keys from translation files :)